math/fibonacci.cpp

```diff
@@ -17,8 +17,9 @@
 unsigned int fibonacci(unsigned int n) {
     /* If the input is 0 or 1 just return the same
        This will set the first 2 values of the sequence */
-    if (n <= 1)
+    if (n <= 1) {
```

Comment:
Shouldn't this file be modified in another PR?

Reply:
Interesting, somehow it got automatically fixed by "clang-tidy fixes for a49ec9b" - see the commit history, not sure I fully understand how the clang-tidy is set up for this repo.

Reply:
This is resolved by clang-tidy, right?

Reply:
I set the clang-tidy up to use the config file `.clang-tidy`. I had initially setup it on all the files but removed it as it was un-necessary. Now, it gets called only on the modified C/CPP files.